go/vt/sqlparser/sql.y

```diff
@@ -1325,6 +1326,10 @@ alter_statement:
   {
     $$ = &DDL{Action: AddVschemaTableStr, Table: $5}
   }
+| ALTER VSCHEMA ADD SEQUENCE TABLE table_name
```

Comment:
Syntactically since we eventually want to support `CREATE SEQUENCE` and not `CREATE SEQUENCE TABLE`, I think this should be `ALTER VSCHEMA ADD SEQUENCE table_name`.

Reply:
Done